app/lib/features/events/pages/event_page.dart

```diff
@@ -167,84 +193,194 @@ class CalendarEventPage extends ConsumerWidget {
-                              const SizedBox(height: 15),
                               Container(
                                 alignment: Alignment.topLeft,
-                                padding: const EdgeInsets.all(8),
-                                child: Text(description),
-                              ),
-                            ],
-                          ),
-                        ),
-                      ),
-                      Flexible(
-                        flex: 1,
-                        child: Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
-                          children: [
-                            DefaultButton(
-                              title: 'Invite',
-                              onPressed: () => onInvite(context),
-                              isOutlined: true,
-                              style: OutlinedButton.styleFrom(
                                 padding: const EdgeInsets.symmetric(
-                                  horizontal: 32,
-                                  vertical: 18,
+                                  horizontal: 15,
+                                  vertical: 5,
                                 ),
-                                side: BorderSide(
-                                  color: Theme.of(context).colorScheme.success,
-                                  width: 1.5,
+                                child: Row(
+                                  mainAxisSize: MainAxisSize.min,
+                                  crossAxisAlignment: CrossAxisAlignment.start,
+                                  children: [
+                                    Text(
+                                      'Time: ',
+                                      style: Theme.of(context)
+                                          .textTheme
+                                          .bodyMedium,
+                                    ),
+                                    const SizedBox(width: 50),
+                                    Flexible(
+                                      flex: 2,
+                                      child: Text(
+                                        time,
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .bodyMedium,
+                                      ),
+                                    ),
+                                  ],
                                 ),
                               ),
-                            ),
-                            PopupMenuButton(
-                              tooltip: 'RSVP',
-                              offset: const Offset(80, 35),
-                              child: Container(
+                              Consumer(
+                                builder: (context, ref, child) {
+                                  // final profile = ref.watch(
```

Comment:
Is this meant to be removed or activated?

Reply:
removed.